cmd/blockchaincmd/deploy.go

```diff
@@ -483,9 +483,11 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 
```

Comment:
you can also add checks to network start and node create, and use the empty string in the json. in some
case we may want to be able to also limit those

Reply:
network start is for local network and we don't have checks for min version for avalanchego

Reply:
addressed for node create

Reply:
can you ask check for min version also in local network? it seems to be also desirable. also good for e2e

Reply:
Do we need it for local network? The main reason it is added for Fuji & Mainnet is because nodes are required to have at least a specific version of avalanchego to interact with these networks. 

In the case of local network, there is no such limitation, so users are free to use whatever version of AvalancheGo that they want (including older versions).

We should have E2E but I think its better to place it for Fuji E2E